packages/devtools_app/lib/src/profiler/flutter/cpu_profile_screen.dart

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import '../../flutter/screen.dart';
+import '../../flutter/table.dart';
+import '../../performance/performance_controller.dart';
+import '../../table_data.dart';
+import '../cpu_profile_columns.dart';
+import '../cpu_profile_model.dart';
+
+class PerformanceScreen extends Screen {
+  const PerformanceScreen() : super('Performance');
+
+  @override
+  Widget build(BuildContext context) => const PerformanceBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return Tab(
+      text: name,
+      icon: Icon(Icons.computer),
+    );
+  }
+}
+
+class PerformanceBody extends StatefulWidget {
+  const PerformanceBody();
+
+  @override
+  PerformanceBodyState createState() => PerformanceBodyState();
+}
+
+class PerformanceBodyState extends State<PerformanceBody> {
+  final PerformanceController _controller = PerformanceController();
+  CpuProfileData data;
+
+  @override
+  void initState() {
```

Comment:
missing a matching 
dispose()

Reply:
As far as I can tell, there's nothing to dispose here.

The performance controller will be garbage collected with this widget, unless it's creating some disposable object we need to tell is about to be GC'd.

Reply:
Yeah just add a TODO to dispose in the future. Looks like it isn't really needed yet.
Technically we would need to make sure that the view wasn't disposed before the 1 second timer goes off but lets not worry about that now and instead worry about it when we have the real pause-resume ui.